### PR TITLE
Remove monkey clothing

### LINKED ISSUE #17680: Monkey with human clothing sprites
## Description

why is this game so baaaaaaaad
**Reproduction**

**Screenshots**
image

### LINKED ISSUE #18692: Monkey wearing a flannel jumpsuit has sprite issues
## Description
Blursed?

**Screenshots**
image
image
image

### LINKED ISSUE #24000: Lack of monkey clothing sprites
For:
- Jumpskirt/qmturtleskirt
- Jumpsuit/kimono
- Jumpsuit/loungewear
- Jumpsuit/ninja
- clothing for NT vs Syndi gamemode
- Jumpsuit/qmturtle
- Jumpsuit/web
- Jumpsuit/flannel.rsi

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

Completely remove monkey clothing exept bartender monkey suit 
fixes #24000 
fixes #17680 
fixes #18692 

## Why / Balance

in anticipation of the voxes, it is better to maintain sprites for them than for monkeys, when the chance that a monkey will wear certain suit is small

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Monkeys and kobolds can no longer wear all types of jumpsuits.

### CODE REVIEW

**Comment by UbaserB** [comment]
Thank you, my god.

File: Resources/Textures/Clothing/Uniforms/Jumpsuit/ce_nt.rsi/meta.json
Comment (lzk228) [inline Resources/Textures/Clothing/Uniforms/Jumpsuit/ce_nt.rsi/meta.json:1]: lmao why is that one had copyright for monkey but no state

**Comment by Dutch-VanDerLinde** [comment]
why are you removing the EVA and hardsuit sprites

**Comment by Dutch-VanDerLinde** [comment]
this just makes monkey unable to survive in space

**Comment by lzk228** [comment]
ok i think we can left eva suits

**Comment by K-Dynamic** [comment]
Why not just keep the sprites in and depreciate them later in favour of displacement-mapped sprites when they need to be updated

**Comment by deathride58** [comment]
It'd make more sense for clothing to simply be restricted if there's no specific sprites available, similar to how it's handled in SS13. It's surprising that isn't already the case.

**Comment by beck-thompson** [comment]
I agree with deathride as well! I feel like a system that restricts clothing or something is better than this temporary fix.

**Comment by Dutch-VanDerLinde** [comment]
probably should also keep the operative jumpsuit and skirt sprites since syndie monkeys use those

**Comment by MACMAN2003** [comment]
> It'd make more sense for clothing to simply be restricted if there's no specific sprites available for a given bodytype, similar to how it's handled in SS13. It's surprising that isn't already the case.

i agree with this, alongside making monkeys compatible with displacement maps later down the line

**Comment by Froozigiusz** [comment]
Maintainare tries not to needlesly remove a part of a game challange (impossible)

Why are you so keen on that? I understand bloat can be scary, but that right here s just silly and dum

**Comment by metalgearsloth** [comment]
I think best thing is just merge this into whatever displacement map pr and then remove them all then rather than doing one then the other in a couple months.

**Pull request closed without merging**